Earthfile

```diff
@@ -115,7 +115,7 @@ test-integration:
   COPY +go/translate /translate
   COPY --dir migrate/mysql migrate
   WITH DOCKER --compose compose.yaml --service mysql --pull migrate/migrate:v$migrate_version --pull golang:$go_version-alpine
-    RUN --no-cache \
+    RUN \
       --secret=googletranslate_account_key \
```

Comment:
how about mounting a secret, if we use file anyways?
```bash
RUN \
 --mount type=secret,target=/translate/google_account_key.json,id=googletranslate_account_key ...
```

the only problem, that I see is in the earthly cloud secrets token is encoded in b64